lib/grammars.coffee

```diff
@@ -253,6 +253,11 @@ module.exports =
       command: "rails"
       args: (context) -> ['runner', context.filepath]
 
+  Rust:
+    "File Based":
+      command: "bash"
+      args: (context) -> ['-c', "rustc " + context.filepath + " -o /tmp/rs.out && /tmp/rs.out"]
```

Comment:
Hi, it does work. It's based on xcrun definitions for c/c++. It doesn't exclude win32 because, unlike xcrun, bash can be installed on win32 and it should work. It's wrapped as single command to bash to invoke two commands as single one - compilation and actual execution of the binary.

It's true, that it's not perfect - in the same way c/c++/objective-c definitions are not perfect. They don't quote file paths properly so if there's a space in the real path, it'll fail. That's the problem with all of those definitions. Also they use hacky way of invoking two commands as one by wrapping everything as bash one-liner.

If you agree to merge it I can send another patch with fixes for those definitions:
- proper escaping of shell params, and
- support for multiple command invocation (for compile-and-run languages) so we don't need to go through bash.